### PR TITLE
CA-344424: ensure we don't get division by 0

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
Another coverity find.

Signed-off-by: Mark Syms <mark.syms@citrix.com>

### CODE REVIEW

File: cpumond/cpumond.c [hunk 1]
```diff
@@ -170,6 +171,9 @@ int cpumond_loop(cpumond_entry_t *cpumond_entry){
         if (statread(statfd, &total2, &idle2) != 0)
             goto out;
 
+	if (unlikely(total2 == total1))
```
Comment (MarkSymsCtx) [inline cpumond/cpumond.c:174]: Actually, this should go to the sleep call

**Review by TimSmithCtx — APPROVED** [review]

File: cpumond/cpumond.c [hunk 2]
```diff
@@ -170,6 +171,11 @@ int cpumond_loop(cpumond_entry_t *cpumond_entry){
         if (statread(statfd, &total2, &idle2) != 0)
             goto out;
 
+	if (unlikely(total2 == total1)) {
+		sleep(1);
```
Comment (BenSimsCitrix) [inline cpumond/cpumond.c:175]: Don't you need to reread total2?

Comment (BenSimsCitrix) [reply cpumond/cpumond.c:175]: Ahh i see now this is within a loop.

**Review by BenSimsCitrix — APPROVED** [review]
LGTM

File: cpumond/cpumond.c (hunk 2 above)
Comment (MarkSymsCtx) [reply cpumond/cpumond.c:175]: Yes, and I used continue with a second sleep as wrapping the update code in a ```if (total2 > total2) {``` still resulted iin Coverity moaning about untrusted divisors :( stupid tool at times.

**Pull request merged**